server/apps/microservices/src/processors/thumbnail.processor.ts

```diff
@@ -51,62 +51,64 @@ export class ThumbnailGeneratorProcessor {
     const jpegThumbnailPath = resizePath + originalFilename + '.jpeg';
 
     if (asset.type == AssetType.IMAGE) {
-      sharp(asset.originalPath)
+      const res = await sharp(asset.originalPath)
```

Comment:
Do we need `res` here? It doesn't seem to be used anywhere.

Reply:
just for debug, i will remove it soon

Reply:
I got you